README.md

```diff
@@ -1,30 +1,9 @@
 # purescript-web-router
 
-A basic pushstate router for React with support for asynchronous routing logic, built using [react-basic-hooks](https://github.com/spicydonuts/purescript-react-basic-hooks). I recommend [routing-duplex](https://github.com/natefaubion/purescript-routing-duplex) for easy parsing and printing.
+A basic pushstate router for React with support for asynchronous routing logic. I recommend [routing-duplex](https://github.com/natefaubion/purescript-routing-duplex) for easy parsing and printing.
```

Comment:
```suggestion
A basic pushstate router with support for asynchronous routing logic. I recommend [routing-duplex](https://github.com/natefaubion/purescript-routing-duplex) for easy parsing and printing.
```

Reply:
The readme looks... a little different now 🙂

Reply:
README looks great :-)

Do you think that making a README part of a "test suite" (so we verify whether it still compiles) is feasible and desired?

I mean I use this strategy across my libs just by using something like: 
https://github.com/paluh/purescript-homogeneous/blob/master/package.json#L10 (`litps` is really a tool written by [@Thimoteus](https://github.com/Thimoteus/literate-purescript/issues/2)).

Reply:
If PS supported literate code I'd consider it, but I don't think it's worth the extra effort. The example project using React has also been updated and it compiles, so most of the code in the updated readme was tested there first.

Some unit tests would be good, but can come a bit later.

Reply:
This `litps` provides support for "literate PS". Maybe I will play around that idea in my... spare time ;-)